gvr-3dcursor-simple/app/src/main/java/org/gearvrf/io/cursorsimple/CursorMain.java

```diff
@@ -63,10 +64,25 @@ public void onInit(GVRContext gvrContext) {
         mainScene = gvrContext.getNextMainScene();
         mainScene.getMainCameraRig().getLeftCamera().setBackgroundColor(Color.BLACK);
         mainScene.getMainCameraRig().getRightCamera().setBackgroundColor(Color.BLACK);
-        cursorManager = new CursorManager(gvrContext, mainScene);
+        List<IoDevice> devices = new ArrayList<IoDevice>();
+
+        //_VENDOR_TODO_ register the devices with Cursor Manager here.
+        /*
+        TemplateDevice device1 = new TemplateDevice(gvrContext, "template_1", "Right controller");
+        TemplateDevice device2 = new TemplateDevice(gvrContext, "template_2", "Left controller");
+        devices.add(device1);
+        devices.add(device2);
+        */
+
+        /*
+        HandTemplateDevice device = new HandTemplateDevice(gvrContext, mainScene);
+        devices.addAll(device.getDeviceList());
```

Comment:
I can do that but it just increases the number of steps required for integration.